android/app/src/main/AndroidManifest.xml

```diff
@@ -19,7 +19,6 @@
     </intent>
   </queries>
   <application android:name=".MainApplication" android:label="@string/app_name" android:icon="@mipmap/ic_launcher" android:allowBackup="true" android:theme="@style/AppTheme" android:requestLegacyExternalStorage="true">
-    <meta-data android:name="com.google.android.geo.API_KEY" android:value="AIzaSyAqRdA27UKKVQioYLHHGFKejiaZdTiaz8k"/>
```

Comment:
Need to keep the maps api key

Reply:
We should be hiding it in some gitignored `.properties` file. But we'll have to make it so that prebuild will generate that similar to the `injectNativePod.js` solution.

Reply:
I'll make an issue so we don't forget https://github.com/tifapp/FitnessProject/issues/399